plugins/org.yakindu.sct.generator.c/src/org/yakindu/sct/generator/c/extensions/Naming.xtend

```diff
@@ -241,12 +241,14 @@ class Naming {
 	}
 	
 	def dispatch scopeTypeDeclMember(EventDefinition it) {
-		if(useOutEventObservables && direction == Direction.OUT) {
-			return '''«OBSERVABLE_TYPE» «eventName»;'''
-		}
 		'''
-		«BOOL_TYPE» «eventRaisedFlag»;
-		«IF type !== null && type.name != 'void'»«typeSpecifier.targetLanguageName» «eventValueVariable»;«ENDIF»
+		«IF useOutEventObservables && direction == Direction.OUT»
+			«OBSERVABLE_TYPE» «eventName»;
+		«ENDIF»
+		«IF (useOutEventObservables && direction == Direction.OUT) || direction == Direction.IN || direction == Direction.LOCAL»
```

Comment:
must be:  IF (useOutEventGetters ...
currently _raised members are only generated if observables is active. Thus code cannot be compiled if setting is: observables = false getter = true